### DISCUSSION TITLE
Shadowtls مشکل

### DISCUSSION THREAD

**Discussion by Evolve6996** [hiddify/Hiddify-Manager#3292] [Q&A سوالات رایج]
سلام دوستان اکثر کانفیگ ها کار میکنن ولی ولی شدو تی ال اس کار نمیکنه از کجا باید ارروش رو پیدا کنم توی لاکش هیچی نبود جز این
2023-10-09T10:27:27.401388Z ERROR shadow_tls: cannot read config file: key must be a string at line 12 column 5

**Comment by Evolve6996** [comment]
دوستان واسه شما این پروتکل کار میکنه ؟

**Reply by F4r3in** [reply]
سلام، نه برای منم کار نمیکنه. مشکل از نوع رمزنگاری ای هست که هیدیفای برای کانفیگ های شادوساکس تنظیم کرده. اگه بشه نوع رمزنگاری رو به chacha20 تغییر داد احتمالا بشه از این پروتوکل هم استفاده کرد. ولی متاسفانه نمیدونم چطوری باید تغییرش بدم.

**Reply by Evolve6996** [reply]
فکر نکنم shadowtls گزینه رمزنگاری داشته باشه
برای تغییرات فکر کنم میتونید برید به فولدرزیر در سرورتون
/opt/hiddify-manager/singbox/configs
هسته سرور رو هم باید روی سینگ باکس بزارید
